test/libyul/yulSyntaxTests/eof/eof_identifiers_not_defined_in_legacy.yul

```diff
@@ -0,0 +1,32 @@
+object "a" {
+    code {
+        auxdataloadn(0)
+        dataloadn(0)
+        eofcreate("name", 0, 0, 0, 0)
+        returncontract("name", 0, 0)
+        rjump()
+        rjumpi()
+        callf(0)
+        jumpf(0)
+        retf()
+        extcall(0, 1, 2, 3)
+        extstaticcall(0, 1, 2)
+        extdelegatecall(0, 1, 2)
+    }
+}
+
+// ====
+// bytecodeFormat: legacy
+// ----
+// DeclarationError 4619: (32-44): Function "auxdataloadn" not found.
+// DeclarationError 4619: (56-65): Function "dataloadn" not found.
+// DeclarationError 4619: (77-86): Function "eofcreate" not found.
+// DeclarationError 4619: (115-129): Function "returncontract" not found.
```

Comment:
`auxdataloadn`, `eofcreate`, `returncontract` should also get an error saying they're only available in EOF.

Reply:
It has to be handled separately in `AsmAnalysis.cpp`. It doesn't pass `validateInstructions` check because they are builtins. I will add some additional `if` to handle these too.